modules/admin_manual/pages/maintenance/migrating_to_kiteworks.adoc

```diff
@@ -160,14 +160,19 @@ For ease of migrating external mounts, the admin should:
 
 === Kiteworks
 
-* As a major prerequisite, the Kiteworks instance *must* be running version 8.6 or higher.
+* The following major prerequisites apply:
+
+** The Kiteworks instance *must* be running version 8.6 or higher.
+** Kiteworks requires a `Data migrator` license.
 
 * You need to login into the Kiteworks appliance as role *System Admin*.
 // The kiteworks satellite service must be activated and available to the system admin user account.
 
 * The Kiteworks system must provide sufficient disk space for the data to be migrated. The ownCloud xref:migration-verification[occ migrate:to-kiteworks:verify] step will report the estimated disk space needed.
 
-* Ensure sufficient quota settings in the Kiteworks user profiles.
+* Ensure quota is set to unlimited in the Kiteworks user profiles.
+
+* Set the web application firewall (WAF) to report only and not to block file upload during migration. Kiteworks support will help to configure the WAF.
```

Comment:
```suggestion
* Set the web application firewall (WAF) to `report only` so that file uploads are not blocked during migration. Kiteworks support will help to configure the WAF.
```